# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kbd/bg_keymaps.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./data/keymaps/i386/qwerty/bg_bds-cp1251.map.orig	2002-10-14 02:39:32.000000000 +0300
+++ ./data/keymaps/i386/qwerty/bg_bds-cp1251.map	2004-01-13 14:05:59.000000000 +0200
@@ -1,12 +1,12 @@
-###  Ver 0.2 - 20021013 
+###  Ver 1.0 - 20040109
 ###  Linux console Bulgarian keymap, BDS (Bulgarian National Standart) cyrillic layout.
 ###  Charset: CP1251 
 ###  Author: Peter Georgiev <peterg@mail.bg>
 ###
 ###     Temporary switching between CYR and LAT is done with AltGr (R_Alt) key or the L_WIN key. 
 ###     AltGr (R_Alt), kakto i L_WIN prevkliuchvat vremenno mezhdu CYR i LAT.                                                                     
-###     Permanent switching between CYR and LAT is done with Ctrl+Shift, or the WIN_MENU key.
-###     Ctrl+Shift, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
+###     Permanent switching between CYR and LAT is done with Left Shift+Alt, or the WIN_MENU key.
+###     Levi Shift+Alt, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
 ###
 ###	Compose key is R_WIN key.
 ###
@@ -65,6 +65,7 @@
 	shift	control	keycode   8 = Control_underscore
 	altgr	control	keycode   8 = Control_underscore
 	shift	altgr	control	keycode   8 = Control_underscore
+	altgr	control	alt	keycode	  8 = ampersand
 keycode   9 = eight           
 	shift	keycode   9 = asterisk        
 	shift	altgr	keycode   9 = slash        
@@ -73,7 +74,7 @@
 	altgr	control	keycode   9 = Delete          
 	shift	altgr	control	keycode   9 = Delete          
 keycode  10 = nine             parenleft        nine             underscore      
-keycode  11 = zero             parenright       zero             +0xB9      
+keycode  11 = zero             parenright       zero             0xB9      
 keycode  12 = minus           
 	shift	keycode  12 = underscore      
 	shift	altgr	keycode  12 = I      
@@ -113,13 +114,13 @@
 	altgr	keycode  18 = +0xE5
 	shift	altgr	keycode  18 = +0xC5
 	control	keycode  18 = Control_e       
-	control alt     keycode  18 = +0x88
+	control alt     keycode  18 = 0x88
 	shift	control	keycode  18 = Control_e       
 	altgr	control	keycode  18 = Control_e       
 	shift	altgr	control	keycode  18 = Control_e       
 	altgr	alt	keycode  18 = Meta_e          
 	shift	altgr	alt	keycode  18 = Meta_E          
-	altgr   control alt     keycode  18 = +0x88 
+	altgr   control alt     keycode  18 = 0x88 
 keycode  19 = +r               
 	shift	keycode  19 = +R               
 	altgr	keycode  19 = +0xE8
@@ -220,8 +221,6 @@
 	altgr	control	alt	keycode  28 = Meta_Control_m  
 	shift	altgr	control	alt	keycode  28 = Meta_Control_m  
 keycode  29 = Control         
-	shift	keycode  29 = AltGr_Lock      
-	shift	altgr	keycode  29 = AltGr_Lock      
 keycode  30 = +a               
 	shift	keycode  30 = +A               
 	altgr	keycode  30 = +0xFC
@@ -326,8 +325,8 @@
 	altgr	alt	keycode  41 = Meta_grave      
 	shift	altgr	alt	keycode  41 = Meta_asciitilde 
 keycode  42 = Shift           
-	control	keycode  42 = AltGr_Lock      
-	altgr	control	keycode  42 = AltGr_Lock      
+	alt	keycode  42 = AltGr_Lock      
+	altgr	alt	keycode  42 = AltGr_Lock      
 keycode  43 = backslash       
 	shift	keycode  43 = bar             
 	altgr	keycode  43 = parenleft
@@ -417,10 +416,10 @@
 	altgr	control	keycode  53 = Delete          
 	shift	altgr	control	keycode  53 = Delete          
 keycode  54 = Shift           
-	control	keycode  54 = AltGr_Lock      
-	altgr	control	keycode  54 = AltGr_Lock      
 keycode  55 = KP_Multiply     
-keycode  56 = Alt             
+keycode  56 = Alt  
+	shift	keycode	 56 = AltGr_Lock
+	shift	altgr	keycode	56 = AltGr_Lock           
 keycode  57 = space           
 	control	keycode  57 = nul             
 	shift	control	keycode  57 = nul             
@@ -630,8 +629,6 @@
 keycode  95 =
 keycode  96 = KP_Enter        
 keycode  97 = Control         
-	shift	keycode  97 = AltGr_Lock      
-	shift	altgr	keycode  97 = AltGr_Lock      
 keycode  98 = KP_Divide       
 keycode  99 = Control_backslash
 keycode 100 = AltGr           
--- ./data/keymaps/i386/qwerty/bg_bds-utf8.map.orig	2002-10-13 20:13:51.000000000 +0300
+++ ./data/keymaps/i386/qwerty/bg_bds-utf8.map	2004-01-13 14:06:18.000000000 +0200
@@ -1,12 +1,12 @@
-###  Ver 0.1 - 20021013 
+###  Ver 1.0 - 20040109 
 ###  Linux console Bulgarian keymap, BDS (Bulgarian National Standart) cyrillic layout.
 ###  Charset: UTF-8 
 ###  Author: Peter Georgiev <peterg@mail.bg>
 ###
 ###     Temporary switching between CYR and LAT is done with AltGr (R_Alt) key or the L_WIN key. 
 ###     AltGr (R_Alt), kakto i L_WIN prevkliuchvat vremenno mezhdu CYR i LAT.                                                                     
-###     Permanent switching between CYR and LAT is done with Ctrl+Shift, or the WIN_MENU key.
-###     Ctrl+Shift, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
+###     Permanent switching between CYR and LAT is done with Left Shift+Alt, or the WIN_MENU key.
+###     Levi Shift+Alt, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
 ###
 ###	Compose key is R_WIN key.
 ###
@@ -65,6 +65,7 @@
 	shift	control	keycode   8 = Control_underscore
 	altgr	control	keycode   8 = Control_underscore
 	shift	altgr	control	keycode   8 = Control_underscore
+	altgr	control	alt	keycode	  8 = ampersand
 keycode   9 = eight           
 	shift	keycode   9 = asterisk        
 	shift	altgr	keycode   9 = slash        
@@ -202,7 +203,7 @@
 	shift	altgr	alt	keycode  26 = Meta_braceleft  
 keycode  27 = bracketright    
 	shift	keycode  27 = braceright      
-	altgr	keycode  27 = colon          
+	altgr	keycode  27 = semicolon          
 	shift	altgr	keycode  27 = U+00A7
 	control	keycode  27 = Control_bracketright
 	shift	control	keycode  27 = Control_bracketright
@@ -220,8 +221,6 @@
 	altgr	control	alt	keycode  28 = Meta_Control_m  
 	shift	altgr	control	alt	keycode  28 = Meta_Control_m  
 keycode  29 = Control         
-	shift	keycode  29 = AltGr_Lock      
-	shift	altgr	keycode  29 = AltGr_Lock      
 keycode  30 = +a               
 	shift	keycode  30 = +A               
 	altgr	keycode  30 = U+044C
@@ -326,8 +325,8 @@
 	altgr	alt	keycode  41 = Meta_grave      
 	shift	altgr	alt	keycode  41 = Meta_asciitilde 
 keycode  42 = Shift           
-	control	keycode  42 = AltGr_Lock      
-	altgr	control	keycode  42 = AltGr_Lock      
+	alt	keycode  42 = AltGr_Lock      
+	altgr	alt	keycode  42 = AltGr_Lock      
 keycode  43 = backslash       
 	shift	keycode  43 = bar             
 	altgr	keycode  43 = parenleft
@@ -417,10 +416,10 @@
 	altgr	control	keycode  53 = Delete          
 	shift	altgr	control	keycode  53 = Delete          
 keycode  54 = Shift           
-	control	keycode  54 = AltGr_Lock      
-	altgr	control	keycode  54 = AltGr_Lock      
 keycode  55 = KP_Multiply     
-keycode  56 = Alt             
+keycode  56 = Alt
+	shift	keycode	 56 = AltGr_Lock          
+	altgr	shift	keycode	56 = AltGr_Lock
 keycode  57 = space           
 	control	keycode  57 = nul             
 	shift	control	keycode  57 = nul             
@@ -630,8 +629,6 @@
 keycode  95 =
 keycode  96 = KP_Enter        
 keycode  97 = Control         
-	shift	keycode  97 = AltGr_Lock      
-	shift	altgr	keycode  97 = AltGr_Lock      
 keycode  98 = KP_Divide       
 keycode  99 = Control_backslash
 keycode 100 = AltGr           
--- ./data/keymaps/i386/qwerty/bg_pho-cp1251.map.orig	2002-10-14 02:38:59.000000000 +0300
+++ ./data/keymaps/i386/qwerty/bg_pho-cp1251.map	2004-01-09 19:11:09.000000000 +0200
@@ -1,12 +1,12 @@
-###  Ver 0.2 - 20021013 
+###  Ver 1.0 - 20040109 
 ###  Linux console Bulgarian keymap, phonetic cyrillic layout.
 ###  Charset: CP1251 
 ###  Author: Peter Georgiev <peterg@mail.bg>
 ###
 ###     Temporary switching between CYR and LAT is done with AltGr (R_Alt) key or the L_WIN key. 
 ###     AltGr (R_Alt), kakto i L_WIN prevkliuchvat vremenno mezhdu CYR i LAT.                                                                     
-###     Permanent switching between CYR and LAT is done with Ctrl+Shift, or the WIN_MENU key.
-###     Ctrl+Shift, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
+###     Permanent switching between CYR and LAT is done with Left Shift+Alt, or the WIN_MENU key.
+###     Levi Shift+Alt, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
 ###
 ###	Compose key is R_WIN key.
 ###
@@ -72,8 +72,8 @@
 	shift	altgr	control	keycode   9 = Delete          
 keycode  10 = nine             parenleft        nine             parenleft       
 keycode  11 = zero             parenright       zero             parenright      
-	control alt     keycode  11 = +0xB9
-	altgr   control alt	keycode	11 = +0xB9	
+	control alt     keycode  11 = 0xB9
+	altgr   control alt	keycode	11 = 0xB9	
 keycode  12 = minus           
 	shift	keycode  12 = underscore      
 	shift	altgr	keycode  12 = underscore      
@@ -113,13 +113,13 @@
 	altgr	keycode  18 = +0xE5
 	shift	altgr	keycode  18 = +0xC5
 	control	keycode  18 = Control_e       
-	control alt     keycode  18 = +0x88
+	control alt     keycode  18 = 0x88
 	shift	control	keycode  18 = Control_e       
 	altgr	control	keycode  18 = Control_e       
 	shift	altgr	control	keycode  18 = Control_e       
 	altgr	alt	keycode  18 = Meta_e          
 	shift	altgr	alt	keycode  18 = Meta_E          
-	altgr   control alt     keycode  18 = +0x88 
+	altgr   control alt     keycode  18 = 0x88 
 keycode  19 = +r               
 	shift	keycode  19 = +R               
 	altgr	keycode  19 = +0xF0
@@ -220,8 +220,6 @@
 	altgr	control	alt	keycode  28 = Meta_Control_m  
 	shift	altgr	control	alt	keycode  28 = Meta_Control_m  
 keycode  29 = Control         
-	shift	keycode  29 = AltGr_Lock      
-	shift	altgr	keycode  29 = AltGr_Lock      
 keycode  30 = +a               
 	shift	keycode  30 = +A               
 	altgr	keycode  30 = +0xE0
@@ -331,8 +329,8 @@
 	altgr	alt	keycode  41 = Meta_grave      
 	shift	altgr	alt	keycode  41 = Meta_asciitilde 
 keycode  42 = Shift           
-	control	keycode  42 = AltGr_Lock      
-	altgr	control	keycode  42 = AltGr_Lock      
+	alt	keycode  42 = AltGr_Lock      
+	altgr	alt	keycode  42 = AltGr_Lock      
 keycode  43 = backslash       
 	shift	keycode  43 = bar             
 	altgr	keycode  43 = +0xFE
@@ -424,10 +422,10 @@
 	altgr	control	keycode  53 = Delete          
 	shift	altgr	control	keycode  53 = Delete          
 keycode  54 = Shift           
-	control	keycode  54 = AltGr_Lock      
-	altgr	control	keycode  54 = AltGr_Lock      
 keycode  55 = KP_Multiply     
 keycode  56 = Alt             
+	shift	keycode	 56 = AltGr_Lock
+	shift	altgr	keycode	 56 = AltGr_Lock
 keycode  57 = space           
 	control	keycode  57 = nul             
 	shift	control	keycode  57 = nul             
@@ -637,8 +635,6 @@
 keycode  95 =
 keycode  96 = KP_Enter        
 keycode  97 = Control         
-	shift	keycode  97 = AltGr_Lock      
-	shift	altgr	keycode  97 = AltGr_Lock      
 keycode  98 = KP_Divide       
 keycode  99 = Control_backslash
 keycode 100 = AltGr           
--- ./data/keymaps/i386/qwerty/bg_pho-utf8.map.orig	2002-10-13 20:13:51.000000000 +0300
+++ ./data/keymaps/i386/qwerty/bg_pho-utf8.map	2004-01-09 19:18:04.000000000 +0200
@@ -1,12 +1,12 @@
-###  Ver 0.2 - 20021013 
+###  Ver 1.0 - 20040109 
 ###  Linux console Bulgarian keymap, phonetic cyrillic layout.
 ###  Charset: UTF-8 
 ###  Author: Peter Georgiev <peterg@mail.bg>
 ###
 ###     Temporary switching between CYR and LAT is done with AltGr (R_Alt) key or the L_WIN key. 
 ###     AltGr (R_Alt), kakto i L_WIN prevkliuchvat vremenno mezhdu CYR i LAT.                                                                     
-###     Permanent switching between CYR and LAT is done with Ctrl+Shift, or the WIN_MENU key.
-###     Ctrl+Shift, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
+###     Permanent switching between CYR and LAT is done with Left Shift+Alt, or the WIN_MENU key.
+###     Levi Shift+Alt, kakto i WIN_MENU prevkliuchvat mezhdu CYR i LAT.           
 ###
 ###	Compose key is R_WIN key.
 ###
@@ -220,8 +220,6 @@
 	altgr	control	alt	keycode  28 = Meta_Control_m  
 	shift	altgr	control	alt	keycode  28 = Meta_Control_m  
 keycode  29 = Control         
-	shift	keycode  29 = AltGr_Lock      
-	shift	altgr	keycode  29 = AltGr_Lock      
 keycode  30 = +a               
 	shift	keycode  30 = +A               
 	altgr	keycode  30 = U+0430
@@ -331,8 +329,8 @@
 	altgr	alt	keycode  41 = Meta_grave      
 	shift	altgr	alt	keycode  41 = Meta_asciitilde 
 keycode  42 = Shift           
-	control	keycode  42 = AltGr_Lock      
-	altgr	control	keycode  42 = AltGr_Lock      
+	alt	keycode  42 = AltGr_Lock      
+	altgr	alt	keycode  42 = AltGr_Lock      
 keycode  43 = backslash       
 	shift	keycode  43 = bar             
 	altgr	keycode  43 = U+044E
@@ -424,10 +422,10 @@
 	altgr	control	keycode  53 = Delete          
 	shift	altgr	control	keycode  53 = Delete          
 keycode  54 = Shift           
-	control	keycode  54 = AltGr_Lock      
-	altgr	control	keycode  54 = AltGr_Lock      
 keycode  55 = KP_Multiply     
 keycode  56 = Alt             
+	shift	keycode	 56 = AltGr_Lock
+	shift	altgr	keycode	 56 = AltGr_Lock
 keycode  57 = space           
 	control	keycode  57 = nul             
 	shift	control	keycode  57 = nul             
@@ -637,8 +635,6 @@
 keycode  95 =
 keycode  96 = KP_Enter        
 keycode  97 = Control         
-	shift	keycode  97 = AltGr_Lock      
-	shift	altgr	keycode  97 = AltGr_Lock      
 keycode  98 = KP_Divide       
 keycode  99 = Control_backslash
 keycode 100 = AltGr           
